onset_maproom/calc.py

```diff
@@ -379,7 +606,39 @@ def seasonal_sum(
     min_count=None,
     time_coord="T",
 ):
-    """Calculates seasonal totals of daily data in season defined by day-month edges"""
+    """ Calculates seasonal totals of daily data in season defined by day-month edges.
```

Comment:
write description in same vein as other seasonal_ functions that use daily_tobegrouped by